### PR TITLE
LFX Spring 22' CI Improvements Blog

### PR DESCRIPTION
**Pull request by PrimalPimmy**
Signed-off-by: PrimalPimmy <Prashant20.pm@gmail.com>

New blog post of my LFX experience! Thanks! This will stay drafted till I get a green light from you folks :rock:

### CODE REVIEW

**Review by darach — COMMENTED** [review]
Love it 👍 A few small tweaks tomorrow and its good to go. 🚀

File: blog/2022-06-03-LFX-Blog-Prashant.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+It was a pleasant night. I was waiting for LFX to send acceptance/rejection e-mails. And there it was, `"Congratulations! You were accepted to CNCF - Tremor"` . It was a great and exciting feeling to start this journey! And here I am, at the end of it, writing this blog. It was wonderful, everything that I expected it to be, and even more so in the 3 months! I am writing this blog about my experience in this mentorship.
+
+## Introduction
+
+My name's Prashant (Also known as Pimmy on the internet), a 2nd-year university student pursuing my Bachelor's degree in Information Technology. This blog will talk about my project experience in contributing to [Tremor](https://www.tremor.rs) as part of LFX Mentorship Program Spring 2022.
+
+## The Problem
+
+We all hate manual tasks, don't we? No seriously if anyone loves doing things on their own, it's totally fine. Ofcourse not everything can be automated. But in this case, it was something more tedious. Here's a flowchart for basic explaination:
```
Comment (skoech) [inline blog/2022-06-03-LFX-Blog-Prashant.md:19]: Of course*
explanation*

File: blog/2022-06-03-LFX-Blog-Prashant.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+![image](https://user-images.githubusercontent.com/23097199/171507584-c8a1bb81-0cbc-4a70-966e-b998758e6430.png)
+
+With all the crates published, including the language-server which follows the exact same process. Tremor has successfully released a new version! Congratulations!
+
+## My thoughts
+
+The tremor community has been extremely helpful in guiding me through the entire mentorship. They have this principle of "Never *worry*, have fun" that will always stay forever with me, and forward in my career. Special mention for [Heinz](https://github.com/Licenser) who mentored me throughout the months and helped me. And to the tremor community in general, my thanks to all of them! 
+I didn't know much about github actions or DevOps in general. But now I can confidently say that I can indeed, _make processes boring by automating them_. I will continue to engage in open source projects, and guide others to the same, cheers!
```
Comment (skoech) [inline blog/2022-06-03-LFX-Blog-Prashant.md:107]: Did you mean less boring? :D

Comment (skoech) [reply blog/2022-06-03-LFX-Blog-Prashant.md:107]: Oh, wow. What a nice blog post! Good work, @PrimalPimmy! :)

Comment (Licenser) [reply blog/2022-06-03-LFX-Blog-Prashant.md:107]: I think boring is exactly the right word here  :D, boring in a good way!

Comment (PrimalPimmy) [reply blog/2022-06-03-LFX-Blog-Prashant.md:107]: Nope, I'm quoting @darach :p

Comment (skoech) [reply blog/2022-06-03-LFX-Blog-Prashant.md:107]: Oh, I didn't get that at all! :D

Comment (PrimalPimmy) [reply blog/2022-06-03-LFX-Blog-Prashant.md:107]: > Oh, wow. What a nice blog post! Good work, @PrimalPimmy! :)

Thanks mate! :rocket:

**Comment by Licenser** [comment]
👍 to the blog great writeup!

**Comment by PrimalPimmy** [comment]
oh no it failed! npm error?

**Comment by darach** [comment]
Looks like a peer dependency error for the mermaid plugin. These issues happen from time to time as dependabot isn't aware of peer dependency constraints

**Comment by Licenser** [comment]
:sob: whyyy!

**Comment by darach** [comment]
I've cleared the netlify cache just in case the npm deps are cached to see if that resolves the build issue. Deploy step running again

**Review by Licenser — APPROVED** [review]

**Pull request merged**